dali/pipeline/data/types_test.cc

```diff
@@ -44,14 +44,27 @@ TYPENAME_FUNC(float);
 TYPENAME_FUNC(double);
 TYPENAME_FUNC(bool);
 
+template <>
+std::string TypesTest<std::vector<uint8>>::TypeName() {
+  return "list of uint8";
+}
+
+template <>
+std::string TypesTest<std::array<std::vector<uint8>, 10>>::TypeName() {
+  return "list of list of uint8";
+}
+
 typedef ::testing::Types<uint8,
                          int16,
                          int32,
                          int64,
                          float16,
                          float,
                          double,
-                         bool> TestTypes;
+                         bool,
+                         std::vector<uint8>,
+                         std::array<std::vector<uint8>, 10>
```

Comment:
Why the magic number 10 here (and above)?

Reply:
Done.